src/main/events/misc/show-open-dialog.ts

```diff
@@ -4,8 +4,10 @@ import { registerEvent } from "../register-event";
 
 const showOpenDialog = async (
   _event: Electron.IpcMainInvokeEvent,
-  options: Electron.OpenDialogOptions
-) => dialog.showOpenDialog(WindowManager.mainWindow, options);
+  options: Electron.OpenDialogOptions,
+) =>
+  WindowManager.mainWindow &&
+  dialog.showOpenDialog(WindowManager.mainWindow, options);
```

Comment:
If this is purely to make tsc happy I would just put an exclamation here.

Reply:
You are spot on